lib/src/screens/send/widgets/send_card.dart

```diff
@@ -669,22 +671,43 @@ class SendCardState extends State<SendCard> with AutomaticKeepAliveClientMixin<S
     _effectsInstalled = true;
   }
 
-  Future<void> _setTransactionPriority(BuildContext context) async {
+  Future<void> pickTransactionPriority(BuildContext context) async {
     final items = priorityForWalletType(sendViewModel.walletType);
     final selectedItem = items.indexOf(sendViewModel.transactionPriority);
+    final customItem = items.firstWhereOrNull(
+        (element) => element == sendViewModel.bitcoinTransactionPriorityCustom);
+    final customItemIndex = customItem != null ? items.indexOf(customItem) : null;
+    final isBitcoinWallet = sendViewModel.walletType == WalletType.bitcoin;
+    double? customFeeRate = isBitcoinWallet ? sendViewModel.customBitcoinFeeRate.toDouble() : null;
```

Comment:
this `firstWhereOrNull` will throw an exception if the wallet is not a bitcoin wallet, and this code can be enhanced and moved to the view model instead of having it in the UI layer

Reply:
fixed